### ISSUE TITLE
Doesn't seem to find tsconfig.json

### ISSUE THREAD

**Issue by dylanparry** [s-panferov/awesome-typescript-loader#216]
I've just tried switching for TS-Loader to Awesome Typescript Loader and have come up against a few issues. I'm using Typescript 1.8.10 and Webpack 1.13.1, so have installed version 1.1.1 of ATL as per the readme.

The errors I'm getting refer to flags that are definitely already set in my `tsconfig.json` file, such as:

```
ERROR in [default] E:/Documents/Visual Studio Projects/Opus4 Core/Opus.Web.Public/scripts/map/bootstrap.tsx:76:13 
Cannot use JSX unless the '--jsx' flag is provided.
```

and:

```
ERROR in [default] E:/Documents/Visual Studio Projects/Opus4 Core/Opus.Web.Public/scripts/map/classes/map-configuration.ts:19:1 
Experimental support for decorators is a feature that is subject to change in a future release. Set the 'experimentalDecorators' option to remove this warning.
```

I’m also seeing errors about being unable to find modules that I’ve installed via node:

```
ERROR in [default] E:/Documents/Visual Studio Projects/Opus4 Core/Opus.Web.Public/scripts/map/bootstrap.tsx:3:23 
Cannot find module 'react'.
```

My `tsconfig.json` is as follows:

```json
{
  "compileOnSave": false,
  "compilerOptions": {
    "target": "es6",
    "module": "commonjs",
    "jsx": "react",
    "sourceMap": true,
    "experimentalDecorators": true
  },
  "awesomeTypescriptLoaderOptions": {
    "useBabel": true,
    "useCache": false,
    "babelOptions": {
      "presets": [
        "es2015"
      ],
      "env": {
        "production": {
          "plugins": [
            "transform-react-constant-elements",
            "transform-react-inline-elements"
          ]
        }
      }
    }
  }
}
```

and my `webpack.config.js` is:

```javascript
var environment = process.env.NODE_ENV || 'development';
environment = environment.trim();
var PRODUCTION = environment === 'production';

var webpack = require('webpack');

var webpackConfig = {
	cache: true,
	entry: {
		'map': './scripts/map/bootstrap.tsx',
	},
	output: {
		path: './wwwroot/scripts',
		filename: '[name].js',
	},
	resolve: {
		extensions: ['', '.js', '.ts', '.tsx'],
	},
	plugins: [
		// Polyfills for fetch and promise
		new webpack.ProvidePlugin({
			'Promise': 'imports?this=>global!exports?global.Promise!es6-promise',
			'fetch': 'imports?this=>global!exports?self.fetch!whatwg-fetch',
		}),
	],
	module: {
		loaders: [
			{
				test: /\.tsx?$/,
				loader: 'awesome-typescript-loader',
			},
		],
	},
};

if (PRODUCTION) {
	// Adds UglifyJS to the plugins
	webpackConfig.plugins.push(
	  new webpack.optimize.UglifyJsPlugin({
	  	sourceMap: false,
	  	compress: {
	  		warnings: false,
	  		sequences: true,
	  		dead_code: true,
	  		conditionals: true,
	  		booleans: true,
	  		unused: true,
	  		if_return: true,
	  		join_vars: true,
	  		drop_console: true,
	  	},
	  	mangle: {
	  		except: ['exports', 'require'],
	  	},
	  	output: {
	  		comments: false,
	  	},
	  })
	);
}

module.exports = webpackConfig;
```

Any idea what I'm doing wrong here?

Thanks,

Dylan
